docs/src/pages/components/flex/react.mdx

```diff
@@ -1,172 +1,51 @@
-import { Flex, Text } from '@aws-amplify/ui-react';
+import { Flex, Text, Button, View } from '@aws-amplify/ui-react';
 import { FlexDemo } from './demo';
-import { Example } from '@/components/Example';
+import { Example, ExampleCode } from '@/components/Example';
+import { DefaultFlexExample } from './examples';
 
-A layout container using Flexbox.
+The Flex primitive provides a Flexbox container with style `display: flex`. To learn how to use Flexbox CSS properties, see the following documentation:
```

Comment:
We literally do not need to separate this, or add it all the way to the `description`

Reply:
Good point! Although, I prefer having 'A layout container using Flexbox' as the main heading because I'd like for our customers to think about the Flex primitive as simply a Flexbox wrapper. The implementation (`display: flex`) and resources for learning Flexbox are details which seem to better fit the description than the heading. Of course, I'm open to whatever we think would make things the most clear to our customers :)